<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54, 54, 54)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ShowPasswordCheck">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>show password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(170, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="fromDateEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgb(85, 85, 85); 
border-radius: 8px; 
padding: 3px; 
background-color: rgb(70, 70, 70); 
color: rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Fetch from:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="allMailCheck">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>fetch all mail</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
